proto/asgt/data/v1/data_service.proto

```diff
@@ -50,8 +52,25 @@ service DataService {
   rpc RegisterQueryStats (RegisterQueryStatsRequest) returns (google.protobuf.Empty);
   
   rpc CallsPerMonthMetric(google.protobuf.Empty) returns (ssn.dataservice.v1.CallsPerMonthResponse);
+  
+  rpc CalculateMetrics(CalculateMetricsRequest) returns (CalculateMetricsResponse);
 }
 
+message CalculateMetricsRequest {
+  int64 start_time = 1;
+  int64 end_time = 2;
+  repeated string tags = 3;
+}
+
+message CalculateMetricsResponse {
+  message MetricRow {
+    string target = 1;
+    google.protobuf.FloatValue confidence = 2;
```

Comment:
Do you foresee more metrics? Planning for the long run it would be easier to make this key/value pairs, but then again this might just be a temporary solution if we are re-engineering the dataservice anyway.

Reply:
there's one more requirement from Claus (https://jira.visma.com/browse/VML-1075)... I tried to put some thought into it  when thinking of what data service should return and I think the histograms should be doable from the information already given from dataservice